ewc-rfc001-issue-verifiable-credential.md

```diff
@@ -137,6 +143,11 @@ For pre-authorised flow, the credential response is as given:
 
 ## 3.3 Discover request
 
+The order is wrong i think.. the correct should be:
```

Comment:
Yes this is correct. I have raised an issue for the same (https://github.com/EWC-consortium/eudi-wallet-rfcs/issues/5). Will you be interested in raising a PR for the same ?